src/loader.ts

```diff
@@ -135,7 +135,7 @@ function getAppWrapperGetter(
       `[qiankun] Wrapper element for ${appName} with instance ${appInstanceId} is not existed!`,
     );
 
-    if (strictStyleIsolation) {
+    if (strictStyleIsolation && supportShadowDOM) {
```

Comment:
```suggestion
    if (strictStyleIsolation) {
```

Do not fix issues more than one in a pr, split the other fixed into another prs